db/data/iana_to_rails_time_zone.yml

```diff
@@ -101,10 +101,10 @@ America/Ciudad_Juarez: America/Denver
 America/Coral_Harbour: America/Bogota
 America/Cordoba: America/Argentina/Buenos_Aires
 America/Costa_Rica: America/Mexico_City
-America/Creston: America/Phoenix  
+America/Creston: America/Phoenix
 America/Cuiaba: America/Caracas
 America/Curacao: America/Puerto_Rico
-America/Danmarkshavn: Etc/Utc
+America/Danmarkshavn: Etc/UTC
```

Comment:
This incorrect value was flagged by one of the new tests. The supported timezone is `Etc/UTC`, [ref](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
```

Failure:
Worldwide::TimeZoneDataConsistencyTest#test_iana_to_rails_time_zone_keys_map_to_an_equivalent_value_when_supported_by_rails [/Users/dominiquefl/src/github.com/Shopify/worldwide/test/worldwide/time_zone_data_consistency_test.rb:37]
Minitest::Assertion: Etc/Utc timezone is not supported by Rails.
```